### DISCUSSION TITLE
把ChatSydney放到服务器上了，开箱即用

### DISCUSSION THREAD

**Discussion by SoraRoseous** [viopsa233/AI-testers-Forum#60] [技术交流]
和PandoraAI一样的使用条件，需要ipv6，建议使用自己的cookie。
地址：https://nekointelligence.cyou:65432/
~话说黑暗大佬能不能解决一下react加载慢的问题，而且后面对话越长就越卡😂~

**Comment by ghost** [comment]
因为当时说不要难用的Node.js，就用不了JS打包工具，只能去esm.sh和skypack国外服务器导入JS模块文件
对话越长越卡可以去掉token计数器

**Comment by ghost** [comment]
对话越长就越卡的问题已经解决，如果还卡请告诉我

**Reply by SoraRoseous** [reply]
好的大佬

**Comment by yinghua8wu** [comment]
我尝试加tls失败了呢，楼主怎么解决的

**Reply by SoraRoseous** [reply]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/54582495/0cf5514e-02e3-4fc9-86f6-8de8a935f4a3)

**Comment by tzcnbo** [comment]
进不去了?

**Reply by SoraRoseous** [reply]
更新PandoraAI中
